app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::BaseController < ApplicationController
+  layout "admin/index"
+  def home
```

Comment:
cách 1 dòng trống ra em 
```suggestion
  def home; end
```